### PR TITLE
refactor: rename HAY and SnBNB to lisUSD and slisBNB

### PR DESCRIPTION
**Pull request by gleiser-oliveira**
## Jira ticket(s)

VEN-2198

## Changes

- Renamed HAY to lisUSD
- Renamed SnBNB to slisBNB

### CODE REVIEW

File: src/packages/tokens/infos/commonTokens/bscMainnet.ts [hunk 1]
```diff
@@ -22,16 +22,16 @@ import ethLogo from 'packages/tokens/img/eth.svg';
...
+import lisusdLogo from 'packages/tokens/img/lisUSD.png';
 import ltcLogo from 'packages/tokens/img/ltc.svg';
 import lunaLogo from 'packages/tokens/img/luna.svg';
 import maticLogo from 'packages/tokens/img/matic.svg';
 import nftLogo from 'packages/tokens/img/nft.png';
 import planetLogo from 'packages/tokens/img/planet.svg';
 import racaLogo from 'packages/tokens/img/raca.png';
 import sdLogo from 'packages/tokens/img/sd.svg';
-import snbnbLogo from 'packages/tokens/img/snBNB.svg';
+import slisbnbLogo from 'packages/tokens/img/slisBNB.png';
```
Comment (therealemjy) [inline src/packages/tokens/infos/commonTokens/bscMainnet.ts]: Nit: slisBnbLogo

File: src/packages/tokens/infos/commonTokens/bscMainnet.ts [hunk 2]
```diff
@@ -22,16 +22,16 @@ import ethLogo from 'packages/tokens/img/eth.svg';
 import fdusdLogo from 'packages/tokens/img/fdusd.svg';
 import filLogo from 'packages/tokens/img/fil.svg';
 import flokiLogo from 'packages/tokens/img/floki.svg';
-import hayLogo from 'packages/tokens/img/hay.png';
 import linkLogo from 'packages/tokens/img/link.svg';
+import lisusdLogo from 'packages/tokens/img/lisUSD.png';
```
Comment (therealemjy) [inline src/packages/tokens/infos/commonTokens/bscMainnet.ts]: Nit: lisUsdLogo

File: src/packages/tokens/infos/commonTokens/bscTestnet.ts [hunk 3]
```diff
@@ -17,15 +17,15 @@ import dogeLogo from 'packages/tokens/img/doge.svg';
 import ethLogo from 'packages/tokens/img/eth.svg';
 import fdusdLogo from 'packages/tokens/img/fdusd.svg';
 import flokiLogo from 'packages/tokens/img/floki.svg';
-import hayLogo from 'packages/tokens/img/hay.png';
+import lisusdLogo from 'packages/tokens/img/lisUSD.png';
```
Comment (therealemjy) [inline src/packages/tokens/infos/commonTokens/bscTestnet.ts]: Nit: lisUsdLogo

File: src/packages/tokens/infos/commonTokens/bscTestnet.ts [hunk 4]
```diff
@@ -17,15 +17,15 @@ import dogeLogo from 'packages/tokens/img/doge.svg';
...
 import ltcLogo from 'packages/tokens/img/ltc.svg';
 import lunaLogo from 'packages/tokens/img/luna.svg';
 import maticLogo from 'packages/tokens/img/matic.svg';
 import nftLogo from 'packages/tokens/img/nft.png';
 import planetLogo from 'packages/tokens/img/planet.svg';
 import racaLogo from 'packages/tokens/img/raca.png';
 import sdLogo from 'packages/tokens/img/sd.svg';
-import snbnbLogo from 'packages/tokens/img/snBNB.svg';
+import slisbnbLogo from 'packages/tokens/img/slisBNB.png';
```
Comment (therealemjy) [inline src/packages/tokens/infos/commonTokens/bscTestnet.ts]: Nit: slisBnbLogo

**Review by therealemjy — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**